Riku/src/GameLogic/LoadedHookable.cpp

```diff
@@ -0,0 +1,36 @@
+#include "LoadedHookable.h"
+#include "../../Riku/TestMove.h"
+
+
+std::shared_ptr<IMove> LoadedHookable::callFuncWithNoArgs(IHookable& hookable, std::string name)
+{
+    auto func = functions.find(name);
+    if (func == functions.end())
+        return std::shared_ptr<IMove>();
+    return std::make_shared<TestMove>(func->second(hookable));
```

Comment:
trochę nie rozumiem co tu robi TestMove

Reply:
Słusznie. To powinien być IMove, bo tak klasa nie wie jaki typ ruchu zwraca funkcja